content/go/concepts/slices/slices.md

```diff
@@ -0,0 +1,125 @@
+---
+Title: 'Slices'
+Description: 'Slices in Go are similar to arrays, but with the flexibility of being able to change their size.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Slices'
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+**Slices** in Go point to a section of an array. They are similar to arrays, but with the flexibility of being able to change their size. Care needs to be taken when modifying elements in slices that point to an existing array, as the element value will change in the underlying array. Slices have a length and capacity.
+
+The length is the number of elements in the slice and can be found using `len(slice_name)`.
+
+The capacity can be found using `cap(slice_name)`. It is the number of elements in the underlying array from the slice's start index to the end of the array.
+
+## Creating and Initializing Slices
+
+### Declare and Initialize a New Slice
+
+#### Syntax
+
+```pseudo
+new_slice := []<datatype>{<values>}
+```
```

Comment:
It would be good to give a sentence explaining the above, especially the significance of the brackets.

Reply:
@SSwiniarski I added in this sentence:

> Slices can be declared and initialized like arrays, with a data type and a list of values, except without the size specified in the square brackets `[]`.